korman/exporter/convert.py

```diff
@@ -522,6 +559,11 @@ def _(temporary, parent):
             return temporary
 
         def do_pre_export(bo):
+            if bo.type == "ARMATURE":
+                so = self.mgr.find_create_object(plSceneObject, bl=bo)
+                self._export_actor(so, bo)
+                # Bake all armature bones to empties - this will make it easier to export animations and such.
+                self.armature.convert_armature_to_empties(bo, lambda obj: handle_temporary(obj, bo))
```

Comment:
This is where we need to be generator aware, instead of passing down this lambda.